server/src/controllers/Events/resolver.ts

```diff
@@ -348,9 +392,11 @@ ${unsubscribe}
       data.sponsor_ids.map((sponsor_id) => ({
         sponsor_id,
       }));
-    const userEventRoleData: Prisma.user_event_rolesCreateWithoutEventsInput = {
-      users: { connect: { id: ctx.user.id } },
-      role_name: 'organizer',
+
+    const eventUserOrganizer: Prisma.event_usersCreateWithoutEventInput = {
```

Comment:
```suggestion
    const eventUserData: Prisma.event_usersCreateWithoutEventInput = {
```
Nitpick: just to be consistent